src/notelistmodel.cpp

```diff
@@ -328,6 +312,22 @@ bool NoteListModel::isInAllNote() const
             && (m_listViewInfo.parentFolderId == SpecialNodeID::RootFolder);
 }
 
+NodeData &NoteListModel::getRef(int row)
```

Comment:
Is there a better way to do what I'm doing here? 😬 

These two methods are doing the exact same thing...

Reply:
TIL.

Then I guess I'll keep both versions and apply the style you mentioned. Thanks!